scripts/globals/status.lua

```diff
@@ -658,7 +658,7 @@ xi.effect =
     ASYLUM                   = 492,
     SUBTLE_SORCERY           = 493,
     STYMIE                   = 494,
-    -- NONE                       = 495,
+    HYSTERIA                 = 495, -- Used for Hysteroanima to stop after readying a weaponskill with no msg.
```

Comment:
where did this 495 come from? These map to status effects as seen by the client, I'm tempted to go contagion transfer this effect to see if I can even steal it... that said, the client will never actually see this effect. We just need a good effect to use to accomplish this task if we use a status effect, that is.

Reply:
if it's a dummy effect not present in the client it needs a higher ID. (see comment on line 797) we treat those a little diff to prevents problems when SE backfills its placeholders

Reply:
Thanks @TeoTwawki that is super helpful, first time putting in a status effect haha, should we use 804? Or should we use 805 as to not change the placeholder declaration?

Reply:
steal 804, update existing 804's to say 805. The placeholder was sort of a guide for the next person :)

Reply:
Thanks! Just wanted to make sure. :)

Reply:
Moved to 804 and updated 805 to be placeholder